resume/resume.org

```diff
@@ -39,10 +40,16 @@ Jacobi provides a financial modelling platform to a wide variety of businesses a
 Responsibilities
 - Build, manage and maintain Jacobi's Kubernetes infrastructure in AWS and Azure
 - Develop, improve and maintain the ansible and terraform code used to deploy that infrastructure
-- Other duties and support tasks as required.
 
 Achievements
 - Setup Gitlab CI and helped migrate software builds into it
```

Comment:
When I'm reading a resume I tend to favor terse, concrete examples since they don't require a ton of effort to digest, but give me a starting point for further conversation. I'm not sure this is "better", but adding this suggestion for contrast to see if stylistically it's something that appeals to you.

```suggestion
- Rebuilt CI infrastructure using Gitlab CI
```